clients/src/main/java/org/apache/kafka/common/metrics/Metrics.java

```diff
@@ -190,6 +236,10 @@ synchronized void registerMetric(KafkaMetric metric) {
         return this.metrics;
     }
 
+    ConcurrentMap<String, List<Sensor>> childrenSensors() {
+        return childrenSensors;
```

Comment:
Will do. I'll also add a comment that this method should only be used for tests (ideally we'd have an annotation for that).